src/main/java/com/sforce/ws/transport/GaeHttpTransport.java

```diff
@@ -230,7 +232,20 @@ public InputStream getContent() throws IOException {
       }
       in = new ByteArrayInputStream(bytes);
 
-      this.successful = resp.getResponseCode() < 400;
+      this.statusCode = resp.getResponseCode();
+      this.successful = this.statusCode < 400;
+      
+      this.headers = new HashMap<String, Collection<String>>();
+      for (HTTPHeader header : resp.getHeadersUncombined()) {
+          String key = header.getName();
+          String value = header.getValue();
+          if (headers.containsKey(key)) {
+              headers.get(key).add(value);
+          } else {
+              headers.put(header.getName(), Arrays.asList(value));
```

Comment:
Javadoc says:
Returns a fixed-size list backed by the specified array. (Changes to the returned list "write through" to the array.)